### PR TITLE
updated to include missing --stream flags

### PR DESCRIPTION
**Pull request by Akyiaa**
story: https://github.com/galasa-dev/projectmanagement/issues/1430
Recently found some errors with galasactl runs submit commands because the --stream flag was not provided when needed. I've updated the docs to reflect this.

### CODE REVIEW

**Review by techcobweb — DISMISSED** [review]

**Comment by techcobweb** [comment]
Please don't merge until @CaroMac has approved.

**Comment by CaroMac** [comment]
as discussed, the runs prepare and runs submit commands now have different --stream values. need to check this and amend accordingly if required before merging.

**Review by CaroMac — APPROVED** [review]

**Pull request merged**
